Server/serverSlate_test.go

```diff
@@ -1,7 +1,29 @@
 package main
 
-import "testing"
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"testing"
+)
 
-func testServerSlate(t *testing.T) {
-	// TODO: Add serverSlate tests
+func TestCalculateLag(t *testing.T) {
+	file, err := ioutil.ReadFile("ocr-data.json")
+	fmt.Println(err)
```

Comment:
Check err properly
if err != nil {
 t.Fatal("Unable to read ocr-data.json: %v", err)
}

Reply:
done